hostinfo/misc.lua

```diff
@@ -132,5 +135,24 @@ local function asyncSpawn(dataArr, spawnFunc, successFunc, finalCb)
   end)
 end
 
+local function logWarn(errTable)
+  local logpath
+  if los.type() ~= 'linux' then
+    -- TODO: properly implement logging in windows
+    logpath = './'
+  else
+    logpath = '/var/log'
+  end
+  logpath = join(logpath, 'rackspace-monitoring-agent-hostinfo.log')
+  logging.init(logging.StdoutFileLogger:new({
```

Comment:
this is expected... the pipe to jq will try and parse the log message, fail and say there is a json error